app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarUIView.kt

```diff
@@ -67,7 +67,8 @@ class ToolbarUIView(
                 view,
                 ShippedDomainsProvider().also { it.initialize(this) },
                 components.core.historyStorage,
-                components.core.sessionManager
+                components.core.sessionManager,
+                components.core.sessionManager.selectedSession?.id
```

Comment:
@colintheshots this won't fix custom tabs though. I mentioned this in my original PR that we need to pass the sessionId from the bundle to this UI component, but I wasn't sure how to do this with the architecture.

The selectedSession will never be a custom tab session.

Reply:
https://github.com/mozilla-mobile/fenix/pull/333#discussion_r252859732

Reply:
Hmm, that's a good point. I wasn't sure if that was how it would work. We can either pass in the session ID or listen to Changes. It's interesting that the toolbar is fixed to a single session here, since it makes sense to change sessions.